### PR TITLE
FR: Clean + Add Franceinfo

### PR DESCRIPTION
**Pull request by BellezaEmporium**

### CODE REVIEW

File: channels/fr.m3u [hunk 1]
```diff
@@ -309,42 +285,36 @@ http://f24hls-i.akamaihd.net/hls/live/520845/F24_ES_HI_HLS/master.m3u8
...
-#EXTINF:-1 tvg-id="France24French.fr" tvg-name="France 24 | French" tvg-country="FR" tvg-language="" tvg-logo="https://raw.githubusercontent.com/geonsey/Free2ViewTV/master/images/logos/France24.png" group-title="",France 24 | French
-http://static.france24.com:80/live/F24_FR_HI_HLS/live_tv.m3u8
 #EXTINF:-1 tvg-id="France3.fr" tvg-name="France 3" tvg-country="FR" tvg-language="French" tvg-logo="" group-title="",France 3
 http://edge9.iptvnetwork.net/live/france3/playlist.m3u8
 #EXTINF:-1 tvg-id="France4.fr" tvg-name="France 4" tvg-country="FR" tvg-language="French" tvg-logo="" group-title="",France 4
 http://edge9.iptvnetwork.net/live/france4/playlist.m3u8
 #EXTINF:-1 tvg-id="France5.fr" tvg-name="France 5" tvg-country="FR" tvg-language="French" tvg-logo="" group-title="",France 5
 http://edge9.iptvnetwork.net/live/france5/playlist.m3u8
+#EXTINF:-1 tvg-id="Franceinfo.fr" tvg-name="Franceinfo" tvg-country="FR" tvg-language="French" tvg-logo="" group-title="",Franceinfo
+https://linkastream.co/headless?url=https://www.youtube.com/watch?v=cKw79MI2ZLc
```
Comment (Dum4G) [inline channels/fr.m3u:299]: Use https://linkastream.co/headless?url=https://www.youtube.com/channel/UCO6K_kkdP-lnSCiO3tPx7WA/live

File: channels/fr.m3u [hunk 2]
```diff
@@ -309,42 +285,36 @@ http://f24hls-i.akamaihd.net/hls/live/520845/F24_ES_HI_HLS/master.m3u8
...
+#EXTINF:-1 tvg-id="Franceinfo.fr" tvg-name="Franceinfo" tvg-country="FR" tvg-language="French" tvg-logo="" group-title="",Franceinfo
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.youtube.com/watch?v=cKw79MI2ZLc
```
Comment (Dum4G) [inline channels/fr.m3u:301]: Same here

File: channels/fr.m3u [hunk 3]
```diff
@@ -309,42 +285,36 @@ http://f24hls-i.akamaihd.net/hls/live/520845/F24_ES_HI_HLS/master.m3u8
...
-https://live.creacast.com/grandlilletv/smil:grandlilletv.smil/chunklist.m3u8
-#EXTINF:-1 tvg-id="GrandLilleTVHD.fr" tvg-name="Grand Lille TV HD" tvg-country="FR" tvg-language="" tvg-logo="" group-title="",Grand Lille TV HD
+https://edge10.vedge.infomaniak.com/livecast/ik:generation-tv/manifest.m3u8
+#EXTINF:-1 tvg-id="GrandLilleTV.fr" tvg-name="Grand Lille TV" tvg-country="FR" tvg-language="French" tvg-logo="http://www.grandlille.tv/wp-content/uploads/2018/10/grand-lille-info-tv-logo.png" group-title="",Grand Lille TV
+http://str81.creacast.com/grandlilletv/smil:grandlilletv.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="GrandLilleTV.fr" tvg-name="Grand Lille TV" tvg-country="FR" tvg-language="French" tvg-logo="http://www.grandlille.tv/wp-content/uploads/2018/10/grand-lille-info-tv-logo.png" group-title="",Grand Lille TV
+https://live.creacast.com/grandlilletv/smil:grandlilletv.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="GrandLilleTV.fr" tvg-name="Grand Lille TV" tvg-country="FR" tvg-language="French" tvg-logo="http://www.grandlille.tv/wp-content/uploads/2018/10/grand-lille-info-tv-logo.png" group-title="",Grand Lille TV
 http://str81.creacast.com/grandlilletv/high/playlist.m3u8
```
Comment (Dum4G) [inline channels/fr.m3u:315]: Non-adaptive link of the the source above?

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/fr.m3u (hunk 3 above)
Comment (LaneSh4d0w) [reply channels/fr.m3u:315]: Good catch.

File: channels/fr.m3u [hunk 4]
```diff
@@ -225,15 +225,7 @@ https://euronews-euronews-spanish-2.roku.wurl.com/manifest/playlist.m3u8
 #EXTVLCOPT:http-user-agent=Mozilla/5.0 (iPhone; CPU iPhone OS 12_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML,like Gecko) Mobile/15E148
 http://ott-cdn.ucom.am/s89/index.m3u8
 #EXTINF:-1 tvg-id="FashionTV.fr" tvg-name="Fashion TV" tvg-country="FR" tvg-language="" tvg-logo="https://i.imgur.com/HBksX8u.png" group-title="Lifestyle",Fashion TV
-http://entertainment.ashttp9.visionip.tv/live/visiontvuk-entertainment-edgytv-hsslive-25f-16x9-SD/chunklist.m3u8
-#EXTINF:-1 tvg-id="FashionTV.fr" tvg-name="Fashion TV" tvg-country="FR" tvg-language="" tvg-logo="" group-title="Lifestyle",Fashion TV
-http://fash1043.cloudycdn.services/slive/_definst_/ftv_ftv_midnite_k1y_27049_midnite_secr_108_hls.smil/chunklist_b1400000_t64NDgwcA==.m3u8
-#EXTINF:-1 tvg-id="FashionTV.fr" tvg-name="Fashion TV" tvg-country="FR" tvg-language="" tvg-logo="https://static.iptv-epg.com/fr/FashionTV.fr.png" group-title="Lifestyle",Fashion TV
-https://fash1043.cloudycdn.services/slive/_definst_/ftv_ftv_paris_pg_4dg_27027_paris_pg18_188_hls.smil/chunklist_b4700000_t64MTA4MHA=.m3u8
```
[the remark this answers is not in the document]
Comment (LaneSh4d0w) [reply channels/fr.m3u:232]: It's not quite sure whether those channels are "correct" or not. If it was to myself anything coming from Fashion TV would have been on the XXX category...

**Review by Dum4G — APPROVED** [review]

**Pull request merged**